locale/texts.pot

```diff
@@ -3,7 +3,7 @@ msgstr ""
 "Content-Type: text/plain; charset=utf-8\n"
```

Comment:
> The next step was to actually run npm install on Node 20, which prompted many changes to the lockfile to respect the new environment. This command was previously run only on Node 14. ( https://github.com/HathorNetwork/hathor-wallet/commit/813bf7f167319a31521ce0dc4b77d348e6b592da ).

> As a consequence, the pot translation files failed and had to be updated using make update_pot ( https://github.com/HathorNetwork/hathor-wallet/commit/5e362c42b3c2c704cbd1beea562aaa9ebf3ac126 )


I don't understand how one thing can be a consequence of the other, why would the line numbers change here?

Reply:
What was the error?

Reply:
Opened #577 to follow this issue.